developers/weaviate/starter-guides/managing-collections/collections-scaling-limits.mdx

```diff
@@ -143,14 +142,14 @@ Use multi-tenancy when you need to support a large number of tenants and priorit
 
 #### Challenges
 
-- **Access control complexity**: [Fine-grained access control](/developers/weaviate/configuration/authorization) must be implemented to ensure data isolation between tenants.
+- **Access control complexity**: [Fine-grained access control](../../configuration/authorization.md) must be implemented to ensure data isolation between tenants.
```

Comment:
I wonder if this should highlight "uniform data schema and collection configuration" (or - something shorter and catchier) as a challenge, given that we have "customizability" as an advantage.